benchmark/layer_norm_backward.cpp

```diff
@@ -264,6 +264,58 @@ NVFUSER_BENCHMARK_RUN(NvFuserScheduler_LayerNorm_BWD_fp16)
     ->Args({16 * 1024, 1600})
     ->Unit(benchmark::kMicrosecond)
     ->UseManualTime();
+
+// Non-divisible batch split
+NVFUSER_BENCHMARK_DEFINE(
+    NvFuserScheduler_LayerNorm_BWD_nondiv_fp16,
+    setupLayerNorm_BWD,
+    NvFuserScheduler_LayerNorm_BWD,
+    DataType::Half);
+
+NVFUSER_BENCHMARK_DEFINE(
+    NvFuserScheduler_LayerNorm_BWD_nondiv_fp32,
+    setupLayerNorm_BWD,
+    NvFuserScheduler_LayerNorm_BWD,
+    DataType::Float);
+
+auto add_args = [](benchmark::internal::Benchmark* b) {
+  const int64_t batch_size = 16 * 1024;
+  std::vector<std::vector<int64_t>> args;
+  std::vector<int64_t> prime_factors = {
```

Comment:
These cases will be rejected by combined scheduler before this PR due to large persistent buffers.